caas/broker.go

```diff
@@ -180,6 +177,18 @@ type Broker interface {
 
 	// ResourceAdopter defines methods for adopting resources.
 	environs.ResourceAdopter
+
+	// ServicePublicHostPortsGetter provides the API to get service public addresses with port.
+	ServicePublicHostPortsGetter
```

Comment:
This is now not needed right? Can be deleted?

Reply:
sure. deleted from interface, do we need remove it from k8s broker?